official/nlp/data/classifier_data_lib.py

```diff
@@ -1275,6 +1276,46 @@ def convert_single_example(ex_index, example, label_list, max_seq_length,
 
   return feature
 
+class AXgProcessor(DataProcessor):
```

Comment:
what's the difference between AxProcessor and this?

Reply:
Ax task is a GLUE task, and AX-g is a SuperGLUE task